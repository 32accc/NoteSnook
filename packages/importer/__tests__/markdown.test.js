const markdown = require("../providers/markdown");

test("convert evernote note enex file", () => {
  let file = {
	  data:`# Hello world
	  This is a note`,
	  fileName:"world.md",
	  createdAt:0,
	  modifiedAt:0
  }

  expect(markdown.convert([file])).toMatchSnapshot("markdown-file");
});