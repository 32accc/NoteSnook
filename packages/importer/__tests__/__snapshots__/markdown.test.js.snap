// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`convert evernote note enex file: markdown-file 1`] = `
Array [
  Object {
    "content": Object {
      "data": "<h1 id=\\"helloworld\\">Hello world</h1>
<pre><code>  This is a note
</code></pre>",
      "type": "html",
    },
    "dateCreated": 0,
    "dateEdited": 0,
    "title": "world",
  },
]
`;
